include/tvSettings.h

```diff
@@ -2622,6 +2622,76 @@ tvError_t GetLdimZoneShortCircuitStatus(unsigned char* shortcircuit_zone_list, u
  */
 tvError_t GetNumberOfDimmingZones(unsigned int* number_of_dimming_zones);
 
+/**
+ * @brief Sets 2Point Custom Whitebalance
+ *
+ * This function sets Whitebalance for a current picture mode index, primary video source @n
```

Comment:
'White point' instead of 'Whitebalance' would be opt.

Is this applicable for any 'current picture mode' ?  (say, if current picture mode is 'movie'/ theater ?)
Is this always source specific ?  Say, I'm watching HDMI input, would the white point adj be source specific ?
Same question goes for video format as well.  Is this always video format specific ?  Say, when SDR is being watched would this be applicable only for SDR content ?

Reply:
'White point' instead of 'Whitebalance' would be opt.
Do you want to change the API name it self to "WhitePoint" instead of "WhiteBalance" ?

(1)Yes.It will be applicable for all pqmode. If Custom color temp selected for any source/pqmode/format then corresponding value willbe fetch and updated in driver.

(2) Yes Its a source specific. We are having (source*format*pqmode)*WB(RGB Gainoffset) no of slots in driver.
(3) Same as (2)
(4) Same as current WB behavior. Only difference is we have provided additional slot for pqmode/source/format.